views/hoc/AdminCheck.jsx

```diff
@@ -0,0 +1,16 @@
+import { useNavigate, useOutletContext } from 'react-router-dom'
+
+import { routes } from '../routes/routes'
+
+const AdminCheck = ({ children }) => {
+  const admin = 'admin'
```

Comment:
let's define this outside of the component body, or hardcode the string in the condition since it is only used there.

Reply:
[68c51a1](https://github.com/AMP-SCZ/dpdash/pull/730/commits/68c51a17778cbfe24e7595789831fc3b9ca73a37)